tests/integration/test_with_flow.py

```diff
@@ -8,13 +10,13 @@ class MockExecutor(Executor):
         def encode(self, **kwargs):
             pass
 
-    with Flow().add(uses=MockExecutor) as f:
-        f.post(on='/test', inputs=[Document(text='whatever')])
+    MockExec = MockExecutor()
```

Comment:
this `MockExecutor` test should be removed. This is not to be tested in this repo

Reply:
removed